src/ops/cli/terraform.py

```diff
@@ -203,5 +203,6 @@ def run_v2_composition(self, args, terraform_path, composition):
         config['terraform'] = {}
         config['terraform']["path"] = "{}{}".format(terraform_path, composition)
         config['terraform']["variables_file"] = "variables.tfvars.json"
-        config['cluster'] = "auto_generated_" + hashlib.md5(self.cluster_config_path).hexdigest()[:6]
+        cluster_id = hashlib.md5(self.cluster_config_path.encode('utf-8')).hexdigest()[:6]
+        config['cluster'] = "auto_generated_" + cluster_id
```

Comment:
since >3.6 f strings are the python way (PEP498) of concatenating strings:
```config['cluster'] = f"auto_generated_{cluster_id}"```
We should replace all strings concatenations with f strings
Nice reading:
https://realpython.com/python-f-strings/?fbclid=IwAR2aIDmI0eYn1X5YxoJwiUqb-gHE7-xu_9mZYMdvQrNiOep-rJ5_51Wckqs